File: server/src/main/java/de/ingrid/igeserver/db/FindOptions.java
package de.ingrid.igeserver.db;

public class FindOptions {
    public QueryType queryType;
    public String queryOperator;
    public Integer size;
    public String sortField;
    public String sortOrder;
    public boolean resolveReferences;

    public FindOptions() {
        this.queryOperator = "OR";
    }
}

File: server/src/main/java/de/ingrid/igeserver/db/FindOptions.kt
package de.ingrid.igeserver.db

class FindOptions(
        val queryType: QueryType?,
        val queryOperator: String = "OR",
        val size: Int? = null,
        val sortField: String? = null,
        val sortOrder: String? = null,
        val resolveReferences: Boolean = false)